bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/Vcs.scala

```diff
@@ -66,40 +78,48 @@ class Vcs(user: VcsAction[Set[Principal]], server: VcsServer, service: Peer => V
     def filter(eval: MergeEval): Boolean = eval.localUpdate
 
     def update(f: ISPFactory, p: ISPProgram, eval: MergeEval): VcsAction[Unit] =
-      eval.plan.merge(f, p)
+      checkCancel(cancelled) >> eval.plan.merge(f, p)
```

Comment:
> Sequentially compose two actions, discarding any value produced by the first, like sequencing  operators (such as the semicolon) in imperative languages.


`a >> b` is used to get the effect of `a` but without needing the output for `b`.  Here I think it should be the same as `a >>= _ => b` or

````scala
for {
  _  <- a
  b0 <- b
} yield b0
````
It's especially appropriate since `checkCancel` produces a `VcsAction[Unit]`.